traefik.yml

```diff
@@ -44,10 +44,6 @@ entryPoints:
       advertisedPort: 443
 
 
-# https://doc.traefik.io/traefik/routing/entrypoints/#http3
-experimental:
-  http3: true
-
```

Comment:
`http3` is now enabled by default, without the associated experimental.http3 option, which [is now removed](https://doc.traefik.io/traefik/migration/v2-to-v3/#http3).